macros/scripts/audio.asm

```diff
@@ -1,21 +1,42 @@
-musicheader: MACRO
-	; number of tracks, track idx, address
-	dbw ((\1 - 1) << 6) + (\2 - 1), \3
+channel_count: MACRO
+num_channels = \1 - 1
+ENDM
+
+channel: MACRO
+	dn (num_channels << 2), \1 - 1 ; channel id
+	dw \2 ; address
+num_channels = 0
 ENDM
 
 note: MACRO
-	dn (\1), (\2) - 1
+	dn (\1), (\2) - 1 ; pitch, length
+ENDM
+
+dnote: MACRO
```

Comment:
`drum_note` would fit better; and why have this if it's identical to `note`?

Reply:
I've already addressed why it ought to be a separate macro from `note`.
I'm open minded to changing to `drum_note` but `dnote` is simply what it has been called in pokered since 2013:
https://github.com/pret/pokered/commit/8edfcc4b1e36111d400b5f104a04001c66416426#diff-0e161c2847e875f13192c5553aa46d49R3425

Reply:
Changed to `drum_note`